Tests/lib/taskRunner.ts

```diff
@@ -18,7 +18,7 @@ function debug(message) {
 }
 
 export class TaskRunner extends events.EventEmitter {
-	constructor(name: string) {
+	constructor(name: string, ignoreSlashes?: boolean) {
```

Comment:
Yes, this is just for testing purposes where we compare the command line that got generated with the one we expected.